github/resource_github_branch_protection_v3_utils.go

```diff
@@ -351,6 +351,7 @@ func expandRequiredPullRequestReviews(d *schema.ResourceData) (*github.PullReque
 			rprr.DismissStaleReviews = m["dismiss_stale_reviews"].(bool)
 			rprr.RequireCodeOwnerReviews = m["require_code_owner_reviews"].(bool)
 			rprr.RequiredApprovingReviewCount = m["required_approving_review_count"].(int)
+			*rprr.RequireLastPushApproval = m["require_last_push_approval"].(bool)
```

Comment:
Do you mind explaining the difference between the code you've written and doing something like:

```go
            requireLastPushApproval := m["require_last_push_approval"].(bool)
// ...
			rprr.RequireLastPushApproval = &requireLastPushApproval
```

Reply:
The difference is my code will NPE if rprr.RequireLastPushApproval is null, which it is likely to be unless that field is explicitly constructed with the rest of the struct.

I guess the bigger, and slightly out of scope for this PR, question is why is that field a pointer?